### DISCUSSION TITLE
Codeium doesn't seem to be working after nvim.cmp replaced by blink.cmp on V14?

### DISCUSSION THREAD

**Discussion by nolongerhesitate** [LazyVim/LazyVim#5036] [Q&A]
The suggestions are not displayed after updating to V14. 
It was working with nvim.cmp and what should I do that make it working with blink.cmp?
Or I can only use nvim.cmp?

**Comment by folke** [comment]
You just need to enable the supermaven extra.
There's also an extra for nvim-cmp if you prefer that.
Just tested and it all works fine for me.

**Reply by rjshrjndrn** [reply]
Hi @folke can you please tell, why do we need to enable supermaven for codieum?

**Reply by folke** [reply]
codeium if you need codeium of course.

**Reply by rjshrjndrn** [reply]
Enabling Codieum is not showing any suggestions.
<img width="1106" alt="image" src="https://github.com/user-attachments/assets/7883ebe5-0068-4e2d-8f5a-108580be2f3b" />
<img width="499" alt="image" src="https://github.com/user-attachments/assets/cc8f53aa-f24f-41eb-8404-a45475b8f1f3" />

**Reply by rjshrjndrn** [reply]
@folke I can share any additional info if you need.

**Reply by folke** [reply]
Did you do `:Codeium Auth`?

**Reply by rjshrjndrn** [reply]
yes @folke

**Reply by rjshrjndrn** [reply]
I tested with the latest release 14.1.0, still codeium is not working with blink

**Comment by patelharsh9797** [comment]
any way to disable coding.blink & fzf-lua?
i guess that will fix my 90% issue with new updates!

**Reply by dpetka2001** [reply]
Just use the `:LazyExtras` to enable `nvim-cmp` and `telescope` like you want.

**Reply by rjshrjndrn** [reply]
but disabling coding.blink is not working @dpetka2001

**Reply by patelharsh9797** [reply]
> Just use the `:LazyExtras` to enable `nvim-cmp` and `telescope` like you want.

I did enable both but the thing is can't disable coding.blink & editor.fzf and funny part is i did not configure anything for that its probably some lazyvim config backend thingi!!! 

![image](https://github.com/user-attachments/assets/1fb9763e-bcc2-4312-a569-43ebd38f8962)
![image](https://github.com/user-attachments/assets/a2febadb-1eb2-48e3-967f-69c6185d22c6)

**Reply by dpetka2001** [reply]
Put in your `options.lua` file `vim.g.lazyvim_cmp = "nvim-cmp"` and `vim.g.lazyvim_picker = "telescope"`. 

@folke for some reason it seems that when set to `auto` they don't disable `blink` and `fzf-lua` from `:LazyExtras`.

**Reply by folke** [reply]
Yep, indeed. Just fixed that.

**Reply by folke** [reply]
It should have worked for telescope though, just broken for nvim-cmp

**Reply by rjshrjndrn** [reply]
@dpetka2001 thank you. That disabled blink. Now codieum is working
<img width="790" alt="image" src="https://github.com/user-attachments/assets/c48990f4-8b50-40d5-ad73-93b2ed50ce07" />

**Reply by dpetka2001** [reply]
For Telescope it works correctly. It was only `nvim-cmp` and `blink`.

**Reply by patelharsh9797** [reply]
every

> Put in your `options.lua` file `vim.g.lazyvim_cmp = "nvim-cmp"` and `vim.g.lazyvim_picker = "telescope"`.
> 
> @folke for some reason it seems that when set to `auto` they don't disable `blink` and `fzf-lua` from `:LazyExtras`.


works now thnx :)

**Reply by dpetka2001** [reply]
Should also work if you update to latest LazyVim right now. What you put in your `options.lua` is not needed any more. If you enable them via `:LazyExtras` right now they will work correctly. I just tested with a fresh LazyVim installation and enabled them via `:LazyExtras` and they get installed correctly.

**Comment by nolongerhesitate** [comment]
I just tried it; it is working right now.
Happy!
